### ISSUE TITLE
UAC warning when uninstalling for the Update.exe

### ISSUE THREAD

**Issue by lipis** [Squirrel/Squirrel.Windows#656]
![uninstall warning](https://cloud.githubusercontent.com/assets/125676/14356099/c277c7e0-fce3-11e5-99e6-d08bd4f80c0a.png)

Is there anything we can do here?

**Comment by paulcbetts** [comment]
If you're seeing UAC pop up when uninstalling, something has gone wrong on your machine, this doesn't happen with other Squirrel uninstalls

**Comment by lipis** [comment]
Thanks for clarifying it.. It was reported by some user.. so I'm guessing their machine was messed up :)

**Comment by ramondeklein** [comment]
I have also noticed this issue and I tracked it down to the new "Apps & features" screen that is the UWP implementation of "Add & Remove programs" from the old control panel. If you start the uninstall from the new UWP window, then `Update.exe` is started with UAC rights. When I uninstall it using the "old" control panel, then it works just fine. Seems like an issue with the new "Apps & Features" window.
